subjects/countstars/README.md

```diff
@@ -0,0 +1,43 @@
+## CountStars
+
+### Instructions
+Write a function called CountStars that counts the stars up to a number given as an argument.
```

Comment:
highlight the function name, don't just copy paste my suggestions, follow the structure you used before

Reply:
Done